dtbase/backend/api/sensor/routes.py

```diff
@@ -105,16 +99,7 @@ def insert_sensor_readings():
 
     payload = json.loads(request.get_json())
     required_keys = ["measure_name", "sensor_uniq_id", "readings", "timestamps"]
-    for k in required_keys:
-        if not k in payload.keys():
-            return (
-                jsonify(
-                    {
-                        "error": f"Must include '{k}' in POST request to /insert_sensor_readings"
-                    }
-                ),
-                400,
-            )
+    check_keys(payload, required_keys, "/insert_sensor_readings")
```

Comment:
I think you need to change the calls to `check_keys` a bit. Currently if there's an error, it gets returned as a json object, but that's discarded since the caller does nothing with the return value of `check_keys`.